doc/cephadm/host-management.rst

```diff
@@ -175,6 +175,20 @@ Where the force flag when entering maintenance allows the user to bypass warning
 
 See also :ref:`cephadm-fqdn`
 
+Rescanning Host Devices
+=======================
+
+Some servers and external enclosures may not register device removal or insertion with the
+kernel. In these scenarios, you'll need to perform a disk rescan. A disk rescan is typically
+non-disruptive, and can be performed with the following CLI command.::
+
+    ceph orch host rescan <hostname> [--with-summary]
```

Comment:
```suggestion
    ceph orch device rescan <hostname> [--with-summary]
```

would it make more sense for this to be placed under a `device` subcommand?

Reply:
I didn't think so. I can see what your saying but to me the action is against the host not against a device - and thats the way I read the syntax structure of the orch commands.

Reply:
The other thing to consider is the UI to CLI relationship. In the UI you would perform a rescan against the 'host object', so having the CLI do the same thing is more intuitive...at least to me!